swim/state_transitions_test.go

```diff
@@ -109,12 +109,42 @@ func (s *StateTransitionsSuite) TestSuspectBecomesFaulty() {
 	s.Require().NotNil(member, "expected cannot be nil")
 
 	s.stateTransitions.ScheduleSuspectToFaulty(*member)
-	s.NotNil(s.stateTransitions.timer(member.Address), "expected suspicion timer to be set")
+	s.NotNil(s.stateTransitions.timer(member.Address), "expected state transtition timer to be set")
 
 	time.Sleep(5 * time.Millisecond)
 	s.Equal(Faulty, member.Status, "expected member to be faulty")
 }
 
+func (s *StateTransitionsSuite) TestFaultyBecomesTombstone() {
+	s.stateTransitions.timeouts.Faulty = time.Millisecond
+
+	s.m.MakeFaulty(s.suspect.Address, s.suspect.Incarnation)
+	member, _ := s.m.Member(s.suspect.Address)
+	s.Require().NotNil(member, "expected cannot be nil")
+
+	s.stateTransitions.ScheduleFaultyToTombstone(*member)
+	s.NotNil(s.stateTransitions.timer(member.Address), "expected state transtition timer to be set")
+
+	time.Sleep(5 * time.Millisecond)
```

Comment:
I think we have a mock clock in place in ringpop now. Can we use that here instead of sleeping?